core/src/main/resources/hudson/model/Computer/index-top.jelly

```diff
@@ -1,15 +1,12 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout">
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:t="/lib/hudson">
+  <link rel="stylesheet" href="${resURL}/jsbundles/simple-page.css" type="text/css" />
```

Comment:
Is this an unrelated change?

Reply:
You true, it makes no more sense, because there are used t:editableDescription. Of cores aI will remove the line.

Reply:
I think simple-page is used to support the login and user creation pages that do not use the usual layout with header and sidepanel, so it is wrong here.

Reply:
Good, that we see it equal. So I am not sure how shall be solve this kind of style issues:
1. Inline style -- dirty solution
2. use a some global css file - you will lost the overview, where is the style used.
3. create new css files (in this case 3 times) with the same content makes also not really sense.


So can we 'vote' which solution will be better. I vote for 1. Dirty, but ease. I think the "style="margin-bottom: var(--section-padding);" is not so much overhead in the files.

Reply:
If you're just after some spacing see https://weekly.ci.jenkins.io/design-library/Spacing/

Reply:
@timja  grat, this might solve my issue . thx